src/workspacer.Native/Native/WindowsDeferPosHandle.cs

```diff
@@ -70,7 +70,9 @@ public void DeferWindowPos(IWindow window, IWindowLocation location)
             int Width = location.Width + offset.Width;
             int Height = location.Height + offset.Height;
 
-            Win32.DeferWindowPos(_info, window.Handle, IntPtr.Zero, X, Y, Width, Height, flags);
+            var oldLocation = window.Location;
+            if (oldLocation.X != X || oldLocation.Y != Y || oldLocation.Width != Width || oldLocation.Height != Height)
+                Win32.DeferWindowPos(_info, window.Handle, IntPtr.Zero, X, Y, Width, Height, flags);
```

Comment:
This change looks harmless enough in itself, but I'm wondering why it's needed now.

Is this required to support the other changes in this PR?

Reply:
Working with Rider, I found that the window would refresh (fully grey for half a second or so) upon forcing a layout. This prevents unnecessary movement, thus resolving the issue.

Reply:
Sounds good. Could have been it's own PR even. But lets keep it here then :)